### PR TITLE
[Estuary] Fix player controls dialog

### PR DESCRIPTION
**Pull request by Hitcher**
## Description

Fix alignment of the 'Repeat' and 'Random' buttons in the Player Controls dialog as well matching the buttons styles.
Fix artwork to show poster if available.
## Motivation and context

Noticed these problems while checking what this dialog actually did.
## How has this been tested?

Tested locally.
## Screenshots (if appropriate):
Before:
screenshot00003
Buttons not lined up with the rest of the controls.
screenshot00002
No poster for the movie currently playing.

After:
screenshot00001
Correctly aligned buttons with the same style as the rest.
screenshot00005
Poster shows when playing movie.
## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by CrystalP** [comment]
I know so little about skinning that I have to defer to skinners for review.
I could run time test though, how is that dialog triggered?

**Comment by KarellenX** [comment]
I've never seen this dialogue before. How do you access it?

**Comment by Hitcher** [comment]
I only noticed this because I was upgrading my skin to 4k and realised I hadn't added any buttons to my one so I switched over to Estuary to check it out.
I had to look up how to open it as well. I guess it's mainly used for game pads, etc where the control device doesn't have the usual 'Stop', 'Play', etc buttons.

**Review by jjd-uk — APPROVED** [review]
Nice fix, while I was aware of these controls I probably would never have noticed this.

> I guess it's mainly used for game pads, etc where the control device doesn't have the usual 'Stop', 'Play', etc buttons.

It's meant primarily for controlling music I think as fullscreen is not required, so makes less sense for video hence probably why it was thumb only for the texture.

File: addons/skin.estuary/xml/PlayerControls.xml
Comment (jurialmunkey) [inline addons/skin.estuary/xml/PlayerControls.xml:146]: Same here.

File: addons/skin.estuary/xml/PlayerControls.xml
Comment (jurialmunkey) [inline addons/skin.estuary/xml/PlayerControls.xml:148]: Same.

**Review by jurialmunkey — COMMENTED** [review]
Minor comments

Also is the extra 40px padding separating the main controls and repeat/shuffle intentional?

File: addons/skin.estuary/xml/PlayerControls.xml
Comment (jjd-uk) [reply addons/skin.estuary/xml/PlayerControls.xml:142]: We do that everywhere for player control buttons, it's the same in the OSD dialogs as well. Can't remember if there's a specific reason at the moment.

**Comment by KarellenX** [comment]
Thanks @jurialmunkey ctrl-n does show that dialogue.

File: addons/skin.estuary/xml/PlayerControls.xml
Comment (jurialmunkey) [reply addons/skin.estuary/xml/PlayerControls.xml]: If it's the prevailing style then no harm in having it. Just the typo to fix then.

Comment (Hitcher) [reply addons/skin.estuary/xml/PlayerControls.xml]: Typo has be fixed.

**Review by jurialmunkey — APPROVED** [review]
Assuming the additional gap between "next" and "random" is intentional, LGTM.

**Comment by jjd-uk** [comment]
I will give at least 24hrs for any objections before merging either tomorrow or sometime soon after.

**Pull request merged**
